main.go

```diff
@@ -6,6 +6,9 @@ import (
 	"os"
 	"os/signal"
 	"time"
+	
+	"github.com/aws/aws-sdk-go-v2/service/sts"
```

Comment:
I think IAM role used to run this container will need additional privileges to run `GetCallerIdentity`. Could you update README.md where we have IAM policy defined, by adding `sts:GetCallerIdentity` into it?